File: LeetCode-Problems/Easy/Number-of-Vowels-in-String-Halves.java
//Java
class Solution {
    public boolean halvesAreAlike(String s) {
        String a = s.substring(0, s.length()/2).toLowerCase();
        String b = s.substring(s.length()/2).toLowerCase();
        char[] vowels = {'a', 'e', 'i', 'o', 'u'};
        int countA = 0;
        int countB = 0;
        
        for (int i = 0; i < a.length(); i++){
            for (int j = 0; j < vowels.length; j++){
                if (a.charAt(i) == vowels[j]){
                    countA++;
                }
                if (b.charAt(i) == vowels[j]){
                    countB++;
                }
            }
        }
        if (countA == countB){
            return true;
        }
        return false;
    }
}

File: LeetCode-Problems/Easy/Number-of-Vowels-in-String-Halves.py
# Python
class Solution(object):
    def halvesAreAlike(self, s):
        """
        :type s: str
        :rtype: bool
        """
        vowels =  ['a', 'e', 'i', 'o', 'u']
        a = lower(s[0:len(s)/2])
        b = lower(s[len(s)/2:len(s)])
        countA = 0
        countB = 0
        for x in range (0, len(s)):
            if (a[x:x+1] in vowels):
                countA = countA + 1
            if (b[x:x+1] in vowels):
                countB = countB + 1
        if (countA == countB):
            return True
        return False
            
        
